docs/angular-testing-library/api.mdx

```diff
@@ -388,7 +388,7 @@ const {debug} = await render(AppComponent)
 debug()
 ```
 
-### `change`
+### `change` Deprecated, use rerender instead.
```

Comment:
Hi, thanks for the PR!
I'm planning to cut the beta release next weak, what do you think about removing `change` from the docs because of that?

Reply:
Hi thanks for the review :)! Yeah could be an excellent idea. I will take the point and remove this part of documentation ! Thanks again for the review

Reply:
it's done :)